packages/playground/wordpress/src/index.ts

```diff
@@ -2,11 +2,7 @@ import { PHP, UniversalPHP } from '@php-wasm/universal';
 import { joinPaths, phpVar } from '@php-wasm/util';
 import { unzipFile } from '@wp-playground/common';
 export { bootWordPress } from './boot';
-export {
-	getLoadedWordPressVersion,
-	isSupportedWordPressVersion,
-} from './version-detect';
-
+export * from './version-detect';
```

Comment:
No real reason. I find it simpler to use `*` instead of coming back to add more exports. 
The only good reason is that it sometimes takes me far to long to understand why my new function can't be imported 😅

Reply:
Let me know if you have a strong preference, I'm happy to change it.

Reply:
> Let me know if you have a strong preference, I'm happy to change it.

Thanks. It's fine ☺️